classes/controller/FrontController.php

```diff
@@ -857,6 +813,7 @@ public function setMedia()
         $this->addCSS([
             _THEME_CSS_DIR_ . 'theme.css',
             _THEME_CSS_DIR_ . 'custom.css',
+            _THEME_CSS_DIR_.'rtl.css',
```

Comment:
I actually dont know how much style you need to make your website RTL.
There is a body class `lang-rtl` or `lang-ltr`, so you can override your styles, I guess the separate file only make sense if there is a lot to rewrite :confused: